test/cli/test_run.py

```diff
@@ -46,12 +46,20 @@ def test_run(rabbit_config):
     gt.wait()
 
 
-def test_main_with_config(rabbit_config):
+def test_main_with_config(rabbit_config, tmpdir):
+
+    config = tmpdir.join('config.yaml')
+    config.write("""
+        WEB_SERVER_ADDRESS: '0.0.0.0:8001'
+        AMQP_URI: '{}'
+        serializer: 'json'
+    """.format(rabbit_config[AMQP_URI_CONFIG_KEY]))
+
     parser = setup_parser()
     args = parser.parse_args([
         'run',
         '--config',
-        TEST_CONFIG_FILE,
+        config.strpath,
```

Comment:
Hmm. The docs are ambiguous. There is an example of using it that way, but it's not explicit.

I think `.strpath` shows the intent more clearly.